openslides/motions/static/js/motions/site.js

```diff
@@ -1305,10 +1305,14 @@ angular.module('OpenSlidesApp.motions.site', [
             return Boolean(isAllowed);
         };
 
+
         // Inline editing functions
         $scope.inlineEditing = MotionInlineEditing;
         $scope.inlineEditing.init($scope, motion);
 
+        // TODO: properties of inlineEditing are currently not watched continuosly,
```

Comment:
a dirty approach to solve this would be to always have a "save" button and giving a message if nothing has changed at all. But I'd prefer these ng-ifs to work